src/tests/class_tests/openms/data/150516_calibration_concentrations.csv

```diff
@@ -0,0 +1,9273 @@
+150516_CM1_Level2,pyr.pyr_1.Light,pyr.pyr_1.Heavy,0,1,uM,1
+150516_CM1_Level2,r5p.r5p_1.Light,r5p.r5p_1.Heavy,0,1,uM,1
+150516_CM1_Level2,ribflv.ribflv_1.Light,ribflv.ribflv_1.Heavy,0,1,uM,1
+150516_CM1_Level2,ru5p-D.ru5p-D_1.Light,ru5p-D.ru5p-D_1.Heavy,0,1,uM,1
+150516_CM1_Level2,s7p.s7p_1.Light,s7p.s7p_1.Heavy,10,1,uM,1
+150516_CM1_Level2,ser-L.ser-L_1.Light,ser-L.ser-L_1.Heavy,0,1,uM,1
+150516_CM1_Level2,skm.skm_1.Light,skm.skm_1.Heavy,0,1,uM,1
+150516_CM1_Level2,succ.succ_1.Light,succ.succ_1.Heavy,0,1,uM,1
+150516_CM1_Level2,succoa.succoa_1.Light,succoa.succoa_1.Heavy,0,1,uM,1
+150516_CM1_Level2,thr-L.thr-L_1.Light,thr-L.thr-L_1.Heavy,0,1,uM,1
+150516_CM1_Level2,thym.thym_1.Light,thym.thym_1.Heavy,10,1,uM,1
+150516_CM1_Level2,trp-L.trp-L_1.Light,trp-L.trp-L_1.Heavy,0,1,uM,1
+150516_CM1_Level2,tyr-L.tyr-L_1.Light,tyr-L.tyr-L_1.Heavy,0,1,uM,1
+150516_CM1_Level2,udp.udp_1.Light,udp.udp_1.Heavy,0,1,uM,1
+150516_CM1_Level2,udpg.udpg_1.Light,udpg.udpg_1.Heavy,0,1,uM,1
+150516_CM1_Level2,udpglcur.udpglcur_1.Light,udpglcur.udpglcur_1.Heavy,0,1,uM,1
+150516_CM1_Level2,ump.ump_1.Light,ump.ump_1.Heavy,0,1,uM,1
+150516_CM1_Level2,ura.ura_1.Light,ura.ura_1.Heavy,200,1,uM,1
+150516_CM1_Level2,urate.urate_1.Light,urate.urate_1.Heavy,0.4,1,uM,1
+150516_CM1_Level2,uri.uri_1.Light,uri.uri_1.Heavy,20,1,uM,1
+150516_CM1_Level2,utp.utp_1.Light,utp.utp_1.Heavy,4,1,uM,1
+150516_CM1_Level2,xan.xan_1.Light,xan.xan_1.Heavy,4,1,uM,1
+150516_CM1_Level3,23dpg.23dpg_1.Light,23dpg.23dpg_1.Heavy,0,1,uM,1
+150516_CM1_Level3,2mcit.2mcit_1.Light,2mcit.2mcit_1.Heavy,0,1,uM,1
+150516_CM1_Level3,2obut.2obut_1.Light,2obut.2obut_1.Heavy,0,1,uM,1
+150516_CM1_Level3,35cgmp.35cgmp_1.Light,35cgmp.35cgmp_1.Heavy,0,1,uM,1
+150516_CM1_Level3,5oxpro.5oxpro_1.Light,5oxpro.5oxpro_1.Heavy,0,1,uM,1
+150516_CM1_Level3,6pgc.6pgc_1.Light,6pgc.6pgc_1.Heavy,1.6,1,uM,1
+150516_CM1_Level3,accoa.accoa_1.Light,accoa.accoa_1.Heavy,0,1,uM,1
+150516_CM1_Level3,acon-C.acon-C_1.Light,acon-C.acon-C_1.Heavy,0,1,uM,1
+150516_CM1_Level3,actp.actp_1.Light,actp.actp_1.Heavy,0,1,uM,1
+150516_CM1_Level3,ade.ade_1.Light,ade.ade_1.Heavy,8,1,uM,1
+150516_CM1_Level3,adn.adn_1.Light,adn.adn_1.Heavy,40,1,uM,1
+150516_CM1_Level3,adp.adp_1.Light,adp.adp_1.Heavy,0,1,uM,1
+150516_CM1_Level3,adpglc.adpglc_1.Light,adpglc.adpglc_1.Heavy,0,1,uM,1
+150516_CM1_Level3,AICAr.AICAr_1.Light,AICAr.AICAr_1.Heavy,0.4,1,uM,1
+150516_CM1_Level3,akg.akg_1.Light,akg.akg_1.Heavy,0,1,uM,1
+150516_CM1_Level3,ala-L.ala-L_1.Light,ala-L.ala-L_1.Heavy,0,1,uM,1
+150516_CM1_Level3,amp.amp_1.Light,amp.amp_1.Heavy,0,1,uM,1
+150516_CM1_Level3,arg-L.arg-L_1.Light,arg-L.arg-L_1.Heavy,0,1,uM,1
+150516_CM1_Level3,asn-L.asn-L_1.Light,asn-L.asn-L_1.Heavy,0,1,uM,1
+150516_CM1_Level3,asp-L.asp-L_1.Light,asp-L.asp-L_1.Heavy,0,1,uM,1
+150516_CM1_Level3,atp.atp_1.Light,atp.atp_1.Heavy,8,1,uM,1
+150516_CM1_Level3,btn.btn_1.Light,btn.btn_1.Heavy,0,1,uM,1
+150516_CM1_Level3,camp.camp_1.Light,camp.camp_1.Heavy,0,1
```

Comment:
do you need a 10k+ line test file just to test whether the parser works?

Reply:
It is going to be used also on another class, _AbsoluteQuantitationStandards_.
But I could create dummy data inside the tests files, or at least strip this file down.